recipes/breakfest/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omellete
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
++ **First cooked by:**
+[@Karyum](https://github.com/karyum)
+
++ **Date:** Week 5 FACN2
++ **Author of Recipe:**
+#MrsSaliba
```

Comment:
I don't think Mrs Saliba invented omelletes...

Reply:
Mate first of all it's omelettes and not omelletes.
send of all she is the author and not the inventor

Reply:
@Karyum omelette is spelt incorrectly four times in the recipe